src/databricks/sql/thrift_backend.py

```diff
@@ -156,6 +156,8 @@ def __init__(
         )
 
         # Cloud fetch
+        self._use_cloud_fetch = kwargs.get("use_cloud_fetch", True)
```

Comment:
`use_cloud_fetch` is passed as the argument of  `execute_command`, please respect that instead of the args in the constructor.

<img width="474" alt="image" src="https://github.com/user-attachments/assets/c0b2814c-b7f4-4af1-b02c-ed76c38b0093" />

Reply:
@jackyhu-db updated it